storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/AbstractJdbcQueryTemplater.java

```diff
@@ -79,24 +82,13 @@ public String loadEntities(
   public Map<String, Timestamp> getTimestampLimits(String entityTableWithRowCountName) {
```

Comment:
Shall I update the return type as `TimestampLimits` in this PR? Or we should do it in the future PRs